docs/get-started/quickstarts/drive-rover.md

```diff
@@ -21,73 +23,126 @@ date: "2022-12-08"
+{{% /alert %}}
 
-- the [Viam Python SDK](https://python.viam.dev/)
-- the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme)
-- the [Viam Flutter SDK](https://flutter.viam.dev/)
-- the [Viam TypeScript SDK](https://ts.viam.dev/)
-- the [Viam C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk/#readme)
+{{% /tab %}}
+{{< /tabs >}}
 
-{{< alert title="Tip" color="tip" >}}
-If you are [renting your rover](https://app.viam.com/try), we recommend that you get the Viam SDK set up before your reservation starts.
-This way, you can maximize the amount of time you have using the Viam Rover.
+{{< expand "Step 2: Install an SDK" >}}
 
-If you are running out of time during your rental, you can [extend your rover rental](/get-started/try-viam/reserve-a-rover/#extend-your-reservation) as long as there are no other reservations.
-{{< /alert >}}
+Navigate to your machine's **CONNECT** tab.
+Click on any of the listed languages and follow the instructions to install the SDK.
 
-## Connect to your rover
+To install your preferred Viam SDK on your Linux or macOS development machine or [single-board computer](/components/board/), run one of the following commands in your terminal:
 
 {{< tabs >}}
 {{% tab name="Python" %}}
 
-The easiest way to get started writing an application with Viam is to navigate to your [machine's page on the Viam app](https://app.viam.com/robots), select the **CONNECT** tab's **Code sample** page, then select **Python** and copy the boilerplate code.
+If you are using the Python SDK, [set up a virtual environment](/sdks/python/python-venv/) to package the SDK inside before running your code, avoiding conflicts with other projects or your system.
 
-{{% snippet "show-secret.md" %}}
+For macOS (both Intel `x86_64` and Apple Silicon) or Linux (`x86`, `aarch64`, `armv6l`), run the following commands:
+
+```sh {class="command-line" data-prompt="$"}
+python3 -m venv .venv
+source .venv/bin/activate
+pip install viam-sdk
+```
+
+Windows is not supported.
+If you are using Windows, use the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) and install the Python SDK using the preceding instructions for Linux.
+For other unsupported systems, see [Installing from source](https://python.viam.dev/#installing-from-source).
+
+If you intend to use the [ML (machine learning) model service](/services/ml/), use the following command instead, which installs additional required dependencies along with the Python SDK:
+
+```sh {class="command-line" data-prompt="$"}
+pip install 'viam-sdk[mlmodel]'
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {class="command-line" data-prompt="$"}
+go get go.viam.com/rdk/robot/client
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+```sh {class="command-line" data-prompt="$"}
+npm install @viamrobotics/sdk
+```
+
+{{< alert title="Info" color="info" >}}
+The TypeScript SDK currently only supports building web browser apps.
+{{< /alert >}}
+
```

Comment:
```suggestion
```
[opt] This doesn't feel relevant to this quickstart--might throw people off and make them think that TS can't drive the rover in a square